sdk/core/Azure.Core/src/Shared/MessagingClientDiagnostics.cs

```diff
@@ -170,10 +170,10 @@ public void InstrumentMessage(IDictionary<string, object> properties, string act
                     DiagnosticScope.ActivityKind.Producer);
                 messageScope.Start();
 
-                Activity activity = Activity.Current;
+                Activity? activity = Activity.Current;
```

Comment:
Curious - was this able to build without these updates?

Reply:
Ah I guess this is due to the upgrade since the new version has nullable references enabled.